### ISSUE TITLE
`indent`gives strange advice on wrapped arguments

### ISSUE THREAD

**Issue by silverwind** [eslint/eslint#3235]
To keep lines within 80 characters, we wrap function arguments like this:
```js
function x(a,
           b,
           c) {
  console.log(a, b, c);
}
```
In 1.0, it errors on the `console` line with `Expected indentation of 13 characters but found 2`, which seems like a bug to me in the new `indent` rule using these settings:

`indent: [2, 2, {"SwitchCase": 1, "VariableDeclarator": 2}]`

cc: @gyandeeps

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by IanVS** [comment]
Possible duplicate of https://github.com/eslint/eslint/issues/3228?

**Comment by silverwind** [comment]
Could be a seperate issue from the looks of it.

**Comment by targos** [comment]
looks like a duplicate of #3173

**Comment by silverwind** [comment]
Yeah, it is. Thanks for digging that out.
